dist/marionette.carpenter.js

```diff
@@ -2623,9 +2623,9 @@ define('templates/row',[],function(){
     
       _print(_safe('\n'));
     
-      _ref = this.columns;
-      for (_i = 0, _len = _ref.length; _i < _len; _i++) {
-        column = _ref[_i];
+      ref = this.columns;
+      for (i = 0, len = ref.length; i < len; i++) {
+        column = ref[i];
```

Comment:
It also looks like the edits were made to backbone.paginator, which is a dependency of this lib. In order to actually persist these changes, you'd have to fork backbone.paginator, and include the forked version as a dependency.